pyproject.toml

```diff
@@ -38,9 +38,11 @@ snowflake-connector-python = { version = "*", optional = true }
 [tool.poetry.dev-dependencies]
 mysql-connector-python = "*"
 preql = "^0.2.14"
+pandas = "^1.3"
```

Comment:
Why is this here?

Reply:
With a lower version, the databricks connector fails because an old version of pandas (<1.3.*) when using. But maybe it can be removed because if databricks has a good version (2.0.2), pandas should have an appropriate good version as well. I'll check if it can be removed.